src/main/java/org/fundacionjala/sfdc/pageObjects/SalesForceMainTab.java

```diff
@@ -0,0 +1,85 @@
+package org.fundacionjala.sfdc.pageObjects;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * SalesForceMainTab.java
+ * Class of Salesforce main tab that contains all the options.
+ */
+public class SalesForceMainTab {
+    private WebDriver webDriver;
+    private By tabButton;
+    private By accountsLink;
+
+    /**
+     * Constructor.
+     * @param webDriver instance.
+     */
+    public SalesForceMainTab(final WebDriver webDriver) {
+        this.webDriver = webDriver;
+        this.tabButton = By.id("AllTab_Tab");
+        this.accountsLink = By.linkText("Accounts");
+    }
+
+    /**
+     * Getter of tabButton.
+     * @return the button.
+     */
+    public WebElement getTabButton() {
+        return this.webDriver.findElement(this.tabButton);
+    }
+
+    /**
+     * Getter of accountsLink.
+     * @return the link.
+     */
+    public WebElement getAccountsLink() {
+        return this.webDriver.findElement(this.accountsLink);
+    }
+
+    /**
+     * Method to click tabButton.
+     * @param button to click.
+     */
+    public void clickTabButton(final WebElement button) {
+        button.click();
+    }
+
+    /**
+     * Method to click accountsLink.
+     * @param link to click.
+     */
+    public void clickAccountsLink(final WebElement link) {
+        link.click();
+    }
+
+    /**
+     * Method to display all the options in Salesforce.
+     */
+    public void displayOptions() {
+        WebElement tabButton = getTabButton();
+        clickTabButton(tabButton);
+    }
+
+    /**
+     * Method to click on Accounts.
+     */
+    public void goToAccounts() {
+        WebElement accountsLink = getAccountsLink();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "accountsLink" which hides the field declared at line 15. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)